### PR TITLE
Add: NotFound component with cutesy stuff. Associated styling and rou…

### PR DESCRIPTION
**Pull request by jfogiato**
### Files Changed:
`NotFound.js`, `NotFound.css`, `App.js`

### What Has Changed: 
Added the `NotFound` component (saw this name in an article and thought it was cool/fun), added some styling, added some cute/fun blurbs, and added the '*' route to `App`.

### Notes: (includes questions, ideas for the future, anything you want to call out)
None!

### CODE REVIEW

File: src/Components/NotFound/NotFound.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const NotFound = () => {
+  return (
+    <section className='notfound-container'>
+      <h1 className='notfound-header'>Oh no! This page either moved or doesn't exist. Try something else.</h1>
+      <div className='notfound-blurb'>
+        <p>PS - Hope you're doing well. 🤍</p>
+        <p>PPS - Proud of you.</p>
+      </div>
+    </section>
```
Comment (espressoGoddess) [inline src/Components/NotFound/NotFound.js:12]: this is super cute, pulled it down just to see how cute and confirmed, its rad!

File: src/Components/App/App.js [hunk 2]
```diff
@@ -51,6 +52,7 @@ const App = () => {
           return <JournalEntry journal={journal}/>
         }}/>
         <Route exact path='/how-you-felt'><FeelingsLog logs={userLogs}/></Route>
+        <Route path='*'><NotFound /></Route>
```
Comment (espressoGoddess) [inline src/Components/App/App.js:55]: purrrf

**Review by espressoGoddess — APPROVED** [review]

File: src/Components/App/App.js (hunk 2 above)
Comment (KatherineBlaine) [reply src/Components/App/App.js:55]: Perfect!

File: src/Components/NotFound/NotFound.js (hunk 1 above)
Comment (KatherineBlaine) [reply src/Components/NotFound/NotFound.js:12]: Love that supportive error handling

**Review by KatherineBlaine — APPROVED** [review]
Such a cute little feature, adds a lot of thought to our error handling!

File: src/Components/NotFound/NotFound.js (hunk 1 above)
Comment (chrissycooper) [reply src/Components/NotFound/NotFound.js:12]: so sweet, I love it

**Review by chrissycooper — APPROVED** [review]

File: src/Components/App/App.js (hunk 2 above)
Comment (jfogiato) [reply src/Components/App/App.js:55]: Correct! Any URL that we don't have a path for will redirect here!

**Pull request merged**